java/src/jmri/web/server/Bundle.properties

```diff
@@ -26,15 +26,16 @@ LabelReadonlyPower = Disable power control in menus
 ToolTipReadonlyPowerTrue = Power control in web pages will not function
 ToolTipReadonlyPowerFalse = Power control in web pages will display a dialog allowing power to be set
 LabelStartup = Start automatically with application
-LabelDisableFrames = Disable frames
-LabelEnableFrames = Enable frames
+LabelDisableFrames = Disable Frames
```

Comment:
Both the [Microsoft][1] and [Apple][2] Human Interface Guidelines suggest this should be sentence case. Can you please revert the capitalization of "frames" in this label?

[1]: https://msdn.microsoft.com/en-us/library/windows/desktop/dn742401.aspx#labels
[2]: https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/OSXHIGuidelines/TerminologyWording.html#//apple_ref/doc/uid/20000957-CH15-SW4